Automattic/wp-calypso
# Loads Jetpack Products into the selector page

#### Changes proposed in this Pull Request

* Pulls Jetpack Product data into the new plan picker page.

TODO:

- [ ] Adds strings for features to products.
- [ ] Create psuedo-products for Jetpack Backup and Scan.
- [ ] Style header.
- [ ] Card container styles.
- [ ] Load new icons.

#### Screenshots
<img width="1603" alt="Screen Shot 2020-08-06 at 4 24 42 PM" src="https://user-images.githubusercontent.com/1760168/89579214-60dec300-d801-11ea-9c6a-caaec0b0282c.png">


#### Testing instructions



* Set A/B test correctly.
* Load v2 of plans page.
* Observe new product cards.
* Change filters and observe cards react.

Fixes 1169247016322522-as-1187857487117223.
